lib/Elastica/Index.php

```diff
@@ -192,23 +192,6 @@ public function deleteDocuments(array $docs)
     }
 
     /**
-     * Optimizes search index.
-     *
-     * Detailed arguments can be found here in the link
-     *
-     * @param array $args OPTIONAL Additional arguments
-     *
-     * @return array Server response
-     *
-     * @deprecated Replaced by forcemerge
```

Comment:
Which also makes me realise, perhaps we should trigger here a deprecated message?